tox.ini

```diff
@@ -59,8 +59,9 @@ source =
 doctests = True
 exclude = .*/,build/,dist/,instance/,venv/
 hang-closing = False
-max-doc-length = 80
-max-line-length = 80
+max-doc-length = 88
+max-line-length = 88
+extend-ignore = E203
```

Comment:
Could you add a comment for E203? I can never remember my flake8 ignore codes

Reply:
E203 is whitespace before ':'. It conflicts with the default behavior of Black per https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html.

I will add the comment.